Packaging is used with Version, parse and InvalidVersion that does not strictly
need the versions specified in setup.cfg
Author: Viktor Rehnberg
--- ./setup.cfg.orig	2023-05-11 10:30:41.231633193 +0200
+++ ./setup.cfg	2023-05-11 10:33:01.127272708 +0200
@@ -27,7 +27,7 @@
 packages = find:
 include_package_data = True
 install_requires = 
-	packaging>=21.3
+	packaging>=20.9
 	Pillow>=8.0.0
 python_requires = >=3.7
 
